catroid/src/main/java/org/catrobat/catroid/content/bricks/FormulaBrick.java

```diff
@@ -170,4 +170,12 @@ public void storeDataForBackPack(Sprite sprite) {
 	public List<BackPackedVariableData> getBackPackedVariableData() {
 		return backPackedVariableData;
 	}
+
+	public static String getStringFromFloatForBrickView(float value) {
+		if (((value % 1.0) == 0) ? true : false) {
```

Comment:
seriously?
can you please check again what this inline-if ("?" ":") do here...

besides that id prefer 
value == Math.ceil(value)
to make it clear on what this actually does